public/index.html

```diff
@@ -3,15 +3,24 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="shortcut icon" href="favicon.ico" />
+
+    <link rel="shortcut icon" href="./resources/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="./resources/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./resources/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./resources/favicon-16x16.png">
+    <link rel="manifest" href="./resources/site.webmanifest">
+    <link rel="mask-icon" href="./resources/safari-pinned-tab.svg" color="#5bbad5">
+
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <base href="%PUBLIC_URL%/" />
-    <title>Gnosis Safe</title>
+    <title>Safe</title>
 
-    <meta property="og:title" content="Gnosis Safe" />
+    <meta property="og:title" content="Safe" />
     <meta
       property="og:description"
-      content="Gnosis Safe is the most trusted platform to manage digital assets on Ethereum"
+      content="Safe (formerly Gnosis Safe) is the most trusted platform to manage digital assets on Ethereum"
```

Comment:
Should we be including the formerly note?

Reply:
That’s for Google

Reply:
Yes, but is it legally alright to be there?

Reply:
I think you cannot deny it’s the former name. Let’s keep hearsay discussions like this off GitHub.